### PR TITLE
Yaswanth/87270/Theme_UI_Enhanchements

### PR DESCRIPTION
**Pull request by yaswanth-deriv**

### CODE REVIEW

File: src/components/nav.jsx [hunk 1]
```diff
@@ -119,15 +93,15 @@ const NavBar = () => {
...
+
         {login_information.is_logged_in && <li onClick={logout}> Sign Out</li>}
       </ul>
+
+      <div class={styles.theme}>
+          <ThemeToggle />
       {login_information.is_logged_in ? (
         <Button
-          category="secondary"
+          category="secondrary"
```
Comment (farrah-deriv) [inline src/components/nav.jsx]:
```suggestion
          category="secondary"
```

File: src/components/nav.jsx [hunk 2]
```diff
@@ -167,9 +141,12 @@ const ThemeToggle = () => {
         checked={is_light_theme()}
         onChange={toggleThemeHandler}
       />
-      <span class={classNames(styles["slider"], styles["round"])} />
+      {/* <span class={classNames(styles["slider"], styles["round"])}/> */}
```
Comment (farrah-deriv) [inline src/components/nav.jsx]: can we remove this comment?

File: src/components/nav.jsx [hunk 3]
```diff
@@ -167,9 +141,12 @@ const ThemeToggle = () => {
...
+{is_light_theme()?<img src='https://i.ibb.co/7JfqXxB/sunny.png' class={styles.themeLogo}/>:<img src='https://i.ibb.co/FxzBYR9/night.png' class={styles.themeLogo}/>}
```
Comment (farrah-deriv) [inline src/components/nav.jsx]: can we import the images as svg?

**Review by nijil-deriv — CHANGES_REQUESTED** [review]
Let's use the icons we already have in deriv-app.

**Review by farrah-deriv — APPROVED** [review]

File: src/components/nav.jsx (hunk 3 above)
Comment (yaswanth-deriv) [reply src/components/nav.jsx]: Done.Thanks

**Review by likhith-deriv — APPROVED** [review]

**Review by nijil-deriv — APPROVED** [review]

**Pull request merged**